src/plugins/workspace/server/permission_control/client.ts

```diff
@@ -175,4 +215,30 @@ export class SavedObjectsPermissionControl {
       result: hasPermissionToAllObjects,
     };
   }
+
+  public cacheSavedObjects(
```

Comment:
Actually it is not doing `cache`, it is adding a list of objects that are awaiting for cache. Maybe rename to like `addSavedObjectsToCachableList`?